backend/src/main/java/com/cruru/email/controller/response/EmailResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.cruru.email.controller.response;
+
+import java.time.LocalDateTime;
+
+public record EmailResponse(
```

Comment:
```suggestion
public record EmailHistoryResponse(
```

요런 클래스명을 사용하는게 어떨까요!
EmailResponse는 너무 의미가 광범위한 것 같아서요 :>